### ISSUE TITLE
v7 Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by RainOrigami** [Longoon12000/DownloadPavlovMapsFromModIo#5]
```
Pavlov VR Mod Downloader version 7
Found 100 subscribed Pavlov VR mods.
Found 51 installed but not subscribed Pavlov VR mods.
Checking for updates...
Index was outside the bounds of the array.
Press any key to exit.
```

**Comment by RainOrigami** [comment]
Probable cause:
https://github.com/RainOrigami/DownloadPavlovMapsFromModIo/blob/8eca2460a3af86f36af77cd15366b9eafe9bf1dd/Program.cs#L304C22-L304C22

When testing with folder "test 1234" in the mods folder, this error can be replicated:

```
Pavlov VR mods directory found at F:\Mods\Pavlov VR.
Do you want to use this directory? (Y/n):
Found 59 subscribed Pavlov VR mods.
Found 8 installed but not subscribed Pavlov VR mods.
Checking for updates...
Index was outside the bounds of the array.

Press any key to exit.
```

Having non-mod folders in the mods folder is unexpected but should be caught and not cause a crash.

Fix:
https://github.com/RainOrigami/DownloadPavlovMapsFromModIo/commit/73449ba6b94fcd2d1317502442b1bbd725e46e2c#diff-0b69b473fe937040615d69f606751f61ddbc2e3a1849360ff2456c22afe88c0bR306

```
Checking for updates...
The mod directory F:\Mods\Pavlov VR\test 1234 does not seem to be a valid Pavlov VR mod. Skipping.
...normal continuation...
```

Waiting for [confirmation of Voldark](https://old.reddit.com/r/PavlovGame/comments/14dbppy/tool_to_download_and_update_maps_from_modio/k3oxhrx/?context=10000) to release hotfix.

**Comment by RainOrigami** [comment]
Fix will be released as v7.1 HOTFIX. Unable to confirm if it's the same issue due to incomplete information. If others encounter in v7.1, please provide additional details.
